Sources/BraintreeCore/BTGraphQLHTTP.m

```diff
@@ -63,6 +63,14 @@ - (void)handleRequestCompletion:(NSData *)data
 {
     // Network error
     if (error) {
+        [self callCompletionBlock:completionBlock body:nil response:(NSHTTPURLResponse *)response error:error];
+        return;
+    }
+
+    if (data == nil || data.length == 0) {
+        NSError *error = [[NSError alloc] initWithDomain:BTHTTPErrorDomain
+                                                    code:BTHTTPErrorCodeUnknown
+                                                userInfo:@{NSLocalizedDescriptionKey: @"An unexpected error occurred with the HTTP request."}];
         [self callCompletionBlock:completionBlock body:nil response:nil error:error];
```

Comment:
@jaxdesmarais Should I include response here? It's included for other errors below.